src/test/groovy/my_job_test.groovy

```diff
@@ -9,7 +9,7 @@ class my_job_test extends Specification {
         given:
         JobManagement jm = new MemoryJobManagement()
 
-        def testjob = new File('example-jobs/sqs_notifier_job.groovy').text
+        def testjob = new File('example-jobs/sauce_connect_job.groovy').text
```

Comment:
it seems a bit strange that we keep changing this line in each PR to reflect the most recent thing being tested -- or am I missing something?